### ISSUE TITLE
Query variables not being replaced

### ISSUE THREAD

**Issue by gingerwizard** [grafana/clickhouse-datasource#53]
Query variables dont seem to be replaced in latest version 0.9.9.

Example query

```
SELECT
        toStartOfMonth(created_at) AS time,
        lower(repo_name) AS repo,
        count() AS c
    FROM github_events
    WHERE (event_type = 'WatchEvent') AND (toYear(created_at) >= 2015) AND (repo IN
    (
        SELECT lower(repo_name) AS repo
        FROM github_events
        WHERE (event_type = 'WatchEvent') AND created_at >= $__fromTime AND created_at <= $__toTime AND repo_name IN (${repo})
        GROUP BY repo
        ORDER BY count() DESC
        LIMIT 10
    ))
    GROUP BY
        repo,
        time
    ORDER BY time;
```

Notice below that repo is not substituted:

```
SELECT
        toStartOfMonth(created_at) AS time,
        lower(repo_name) AS repo,
        count() AS c
    FROM github_events
    WHERE (event_type = 'WatchEvent') AND (toYear(created_at) >= 2015) AND (repo IN
    (
        SELECT lower(repo_name) AS repo
        FROM github_events
        WHERE (event_type = 'WatchEvent') AND created_at >= toDateTime(intDiv(1485969538965,1000)) AND created_at <= toDateTime(intDiv(1643735938965,1000)) AND repo_name IN (${repo})
        GROUP BY repo
        ORDER BY count() DESC
        LIMIT 10
    ))
    GROUP BY
        repo,
        time
    ORDER BY time;

```

Configuration for query variable:

name: repo
query: `SELECT DISTINCT repo_name FROM github_events WHERE like(repo_name, 'apache%') LIMIT 100`

Credentials for dataset [here](https://ghe.clickhouse.tech/#clickhouse-demo-access)

**Comment by gingerwizard** [comment]
Global variables also not being replaced e.g. replace 2nd line with

`toStartOfInterval(created_at, INTERVAL intDiv(${__interval_ms},1000) second) AS t,`

**Comment by scottlepp** [comment]
Fixed by #59
